packages/rum-core/src/domain/view/viewMetrics/trackScrollMetrics.spec.ts

```diff
@@ -26,117 +59,61 @@ describe('trackScrollMetrics', () => {
+    scrollObservable.notify({ scrollDepth: 700, scrollHeight: 1900, scrollTop: 100 })
+    expect(callbackSpy).toHaveBeenCalledOnceWith({
+      maxDepth: 700,
+      maxDepthScrollHeight: 2000,
+      maxDepthTime: jasmine.any(Number),
+      maxDepthScrollTop: 100,
     })
-
-    it('should NOT update scroll metrics when scroll depth has not increased', () => {
-      newScroll({ scrollHeight: 1000, scrollDepth: 600, scrollTop: 200 })
-
-      newScroll({ scrollHeight: 1000, scrollDepth: 450, scrollTop: 50 })
-
-      expect(scrollMetrics).toEqual({
-        maxDepthScrollHeight: 1000,
-        maxDepth: 600,
-        maxDepthScrollTop: 200,
-        maxDepthTime: 1000 as Duration,
-      })
+    expect(callbackSpy).not.toHaveBeenCalledOnceWith({
+      maxDepth: 700,
+      maxDepthScrollHeight: 1900,
+      maxDepthTime: jasmine.any(Number),
+      maxDepthScrollTop: 100,
     })
   })
 
-  describe('on load', () => {
-    beforeEach(() => {
-      setupBuilder.withFakeClock()
-    })
-
-    it('should have an undefined loading time and empty scroll metrics if there is no activity on a route change', () => {
-      const { clock } = setupBuilder.build()
-      const { getViewUpdate, getViewUpdateCount, startView } = viewTest
-
-      startView()
-      clock.tick(THROTTLE_VIEW_UPDATE_PERIOD)
-
-      expect(getViewUpdateCount()).toEqual(3)
-      expect(getViewUpdate(2).commonViewMetrics.loadingTime).toBeUndefined()
-      expect(getViewUpdate(2).commonViewMetrics.scroll).toEqual(undefined)
+  it('should update max depth only if it has increased', () => {
+    scrollObservable.notify({ scrollDepth: 700, scrollHeight: 2000, scrollTop: 100 })
+    scrollObservable.notify({ scrollDepth: 600, scrollHeight: 2000, scrollTop: 0 })
+    expect(callbackSpy).toHaveBeenCalledOnceWith({
+      maxDepth: 700,
+      maxDepthScrollHeight: 2000,
+      maxDepthTime: jasmine.any(Number),
+      maxDepthScrollTop: 100,
     })
-
-    it('should have a loading time equal to the activity time and scroll metrics if there is a unique activity on a route change', () => {
-      const { domMutationObservable, clock } = setupBuilder.build()
-      const { getViewUpdate, startView } = viewTest
-
-      startView()
-      clock.tick(BEFORE_PAGE_ACTIVITY_VALIDATION_DELAY)
-      domMutationObservable.notify()
-      clock.tick(AFTER_PAGE_ACTIVITY_END_DELAY)
-      clock.tick(THROTTLE_VIEW_UPDATE_PERIOD)
-
-      expect(getViewUpdate(3).commonViewMetrics.loadingTime).toEqual(BEFORE_PAGE_ACTIVITY_VALIDATION_DELAY)
-      expect(getViewUpdate(3).commonViewMetrics.scroll).toEqual({
-        maxDepthScrollHeight: jasmine.any(Number),
-        maxDepth: jasmine.any(Number),
-        maxDepthTime: jasmine.any(Number),
-        maxDepthScrollTop: jasmine.any(Number),
-      })
+    expect(callbackSpy).not.toHaveBeenCalledOnceWith({
+      maxDepth: 600,
+      maxDepthScrollHeight: 2000,
+      maxDepthTime: jasmine.any(Number),
+      maxDepthScrollTop: 0,
     })
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Same here, this assertion is not useful